backend/src/model/dto.rs

```diff
@@ -81,55 +82,84 @@ pub struct PlantsSummaryDto {
 /// Represents plant planted on a map.
 /// E.g. a user drags a plant from the search results and drops it on the map.
 #[typeshare]
-#[derive(Serialize, Deserialize, ToSchema)]
-pub struct PlantingDto {
-    /// The database id of the record.
-    pub id: i32,
+#[derive(Serialize, Deserialize, ToSchema, Clone)]
+pub struct PlantLayerObjectDto {
```

Comment:
Why do we need this? What is the difference to NewPlantingDto?

Reply:
If we add the creation_date and deletion_date fields they will diverge. Those fields are not initialised by the user.